media/static-bundles.json

```diff
@@ -1406,6 +1412,12 @@
         "js/base/error-pages.js"
       ],
       "name": "page_not_found"
+    },
+    {
+      "files": [
+        "js/firefox/installer-help-experiment.js"
```

Comment:
The page is throwing a JS error because Traffic Cop is undefined:

```
Uncaught TypeError: Mozilla.TrafficCop is not a constructor
```

Reply:
whoops! 😅